docker-compose.yml

```
@@ -69,3 +69,24 @@ services:
     volumes:
       - .:/velox:delegated
     command: cd /velox && make python-test
+
+
+  presto-java:
+  # Usage:
+  #   docker-compose pull presto-java or docker-compose build presto-java
+  #   docker-compose run --rm presto-java
+  #   or
+  #   docker-compose run -e NUM_THREADS=<NUMBER_OF_THREADS_TO_USE> --rm presto-java
+  #   to set the number of threads used during compilation
+    image: ghcr.io/facebookincubator/velox-dev:amd64-centos-8-avx
```

```suggestion
    image: ghcr.io/facebookincubator/velox-dev:presto-java
```
This will be the name of the build image :)